pkg/ansible/flags/flag.go

```diff
@@ -34,6 +34,9 @@ type Flags struct {
 	AnsibleVerbosity        int
 	AnsibleRolesPath        string
 	AnsibleCollectionsPath  string
+	EnableLeaderElection    bool
+	LeaderElectionID        string
+	LeaderElectionNamespace string
```

Comment:
Just add the bool at the end to fix the maligned lint check issue.

Reply:
Had to move the booleans together to get the check to pass.